src/Ampersand/PatchHelper/Patchfile/Entry.php

```diff
@@ -390,4 +390,56 @@ public function getRemovedQueueConsumers()
     {
         return $this->getAddedOrRemovedQueueConsumers('original');
     }
+
+    /**
+     * @return array
+     */
+    public function getDatabaseTablesDefinitionsFromOriginalFile()
+    {
+        return $this->getDatabaseTablesDefinitionsFromFile($this->originalFilePath);
+    }
+
+    /**
+     * @return array
+     */
+    public function getDatabaseTablesDefinitionsFromNewFile()
+    {
+        return $this->getDatabaseTablesDefinitionsFromFile($this->newFilePath);
+    }
+
+    /**
+     * @param $file
```

Comment:
Param type missing here

Reply:
Will correct shortly. FWIW this repo needs some serious love https://github.com/AmpersandHQ/ampersand-magento2-upgrade-patch-helper/issues/77 

needs much more static analysis and testing